examples/04_glm_first_level/plot_hrf.py

```diff
@@ -1,65 +1,101 @@
-"""Example of hemodynamic response functions.
-==========================================
+"""
+Example of MRI response functions
+=================================
 
 Within this example we are going to plot the hemodynamic response function
-(hrf) model in SPM together with the hrf shape proposed by G.Glover, as well as
+(HRF) model in SPM together with the HRF shape proposed by G.Glover, as well as
 their time and dispersion derivatives.
+We also illustrate how users can input a custom response function,
+which can for instance be useful when dealing with non human primate data
+acquired using a contrast agent. In our case, we input a custom response function
+for MION, a common agent used to enhance contrast on MRI images of monkeys.
 
-This example requires matplotlib.
-
-The hrf is the filter that couples neural responses to the metabolic-related
-changes in the MRI signal.  hrf models are simply phenomenological.
+The HRF is the filter which couples neural responses to the metabolic-related
+changes in the MRI signal. HRF models are simply phenomenological.
 
-In current analysis frameworks, the choice of hrf model is essentially left to
-the user. Fortunately, using the spm or Glover model does not make a huge
-difference.  Adding derivatives should be considered whenever timing
-information has some degree of uncertainty. It is actually useful to detect
+In current analysis frameworks, the choice of HRF model is essentially left to
+the user. Fortunately, using the SPM or Glover model does not make a huge
+difference. Adding derivatives should be considered whenever timing
+information has some degree of uncertainty, and is actually useful to detect
 timing issues.
 
+This example requires matplotlib and scipy.
 """
+
 #########################################################################
-# Set up some parameters for model display
+# Define stimulus parameters and response models
 # ----------------------------------------
```

Comment:
You need to add some "-" to match the title length otherwise Sphinx will raise an error in strict mode